arrows/vxl/hashed_image_classifier.h

```diff
@@ -0,0 +1,191 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#ifndef HASHED_IMAGE_CLASSIFIER_H_
+#define HASHED_IMAGE_CLASSIFIER_H_
+
+#include <vgl/vgl_box_2d.h>
+#include <vil/vil_image_view.h>
+#include <vil/vil_rgb.h>
+
+#include <fstream>
+#include <iostream>
+#include <limits>
+#include <memory>
+#include <vector>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace vxl {
+
+template < typename FeatureType,
+           typename OutputType > class hashed_image_classifier;
```

Comment:
```suggestion
template < typename FeatureType, typename OutputType >
class hashed_image_classifier;
```

Although, better, I think if you kick the `operator<<`s to the bottom, you don't need the forward declaration at all?

Reply:
I just reformatted this line as it seemed to be breaking initially when I moved the `operator<<`s.